.github/workflows/main.yml

```diff
@@ -36,22 +36,21 @@ jobs:
         run: pip install -r requirements.txt
 
       - name: Run doctests
-        run: python -m doctest -v scripts/header_parser.py
+        run: |
+          python -m doctest -v scripts/header_parser.py
+          python -m doctest -v scripts/expressions.py
 
       - name: Run wrapper generator
-        run: python scripts/header_parser.py C_samples/*.h -vv
+        run: |
+          python scripts/header_parser.py C_samples/*.h -vv
+          python scripts/prepare.py C_samples DESCRIPTION -o out -v 0.1
```

Comment:
What is `-v 0.1` ?

Reply:
this tells the script to replace the `##VERSION##` in DESCRIPTION with `0.1`. It's meant to be backwards compatible with libamtrack's old DESCRIPTION (it can be reused)

Reply:
In that case can you use full names, like `--out-dir` and `--version` ?